test/geometry/test_plane.py

```diff
@@ -49,6 +51,16 @@ def test_smoke(self, device, dtype, shape):
         assert pl0.normal.shape == shape or () + (3,)
         assert pl0.offset.shape == (shape + () if shape is not None else ())
 
+    def test_serialization(self, device, dtype):
```

Comment:
```suggestion
    def test_serialization(self, device, dtype, tmp_path):
```

from [pytest docs](https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html#the-tmp-path-factory-fixture)